pkgs/servers/sql/postgresql/ext/pipelinedb.nix

```diff
@@ -34,6 +34,5 @@ stdenv.mkDerivation rec {
     license = licenses.asl20;
     platforms = postgresql.meta.platforms;
     maintainers = [ maintainers.marsam ];
-    broken = versions.major postgresql.version != "11";
```

Comment:
I think it could better to remove this file, since PipelineDB only supports PostgreSQL 11, and has been abandoned by upstream.

Reply:
(not to self: `== != !=` :upside_down_face: )

Anyways, dropped it now!